api/rds.proto

```diff
@@ -279,7 +279,12 @@ message VirtualCluster {
 
   // Optionally specifies the HTTP method to match on. For example GET, PUT,
   // etc.
-  string method = 3;
+  RequestMethod method = 3;
+
+  message DeprecatedV1 {
```

Comment:
We are not using this internally and I nearly guarantee that no one else is. I would be in favor of just deleting this entire feature, but up to you.